pom.xml

```diff
@@ -39,6 +39,7 @@
 		<dependency>
 			<groupId>com.mysql</groupId>
 			<artifactId>mysql-connector-j</artifactId>
+			<version>8.0.33</version>
```

Comment:
Remind me why this was necessary?

Reply:
because I had a different version :) and not everyone uses the same, so you can setup a one, which most probably everyone can use :) (configuration)